packages/sit-onyx/src/components/OnyxButton/OnyxButton.vue

```diff
@@ -49,24 +49,10 @@ const emit = defineEmits<{
 
 <style lang="scss">
 @use "../../styles/mixins/layers.scss";
-@use "../../styles/mixins/density.scss";
 
 .onyx-button,
 .onyx-button-skeleton {
-  @include density.compact {
-    --onyx-button-height: 2rem;
-    --onyx-button-padding-vertical: var(--onyx-spacing-4xs);
-  }
-
-  @include density.default {
-    --onyx-button-height: 2.5rem;
-    --onyx-button-padding-vertical: var(--onyx-spacing-2xs);
-  }
-
-  @include density.cozy {
-    --onyx-button-height: 3rem;
-    --onyx-button-padding-vertical: var(--onyx-spacing-sm);
-  }
+  --onyx-button-padding-vertical: var(--onyx-density-xs);
```

Comment:
why do you keep `--onyx-button-padding-vertical` unlike in other components?
is there a pattern when you keep it / when you drop it?

Reply:
Because it is used multiple times (for the button and the skeleton).
If a variable is only used once, I removed it, otherwise I kept it